IdentityCore/src/requests/MSIDSilentTokenRequest.m

```diff
@@ -214,7 +222,9 @@ - (void)executeRequestImpl:(MSIDRequestCompletionBlock)completionBlock
             MSID_LOG_WITH_CTX(MSIDLogLevelInfo, self.requestParameters, @"Access token has expired, but it is long-lived token.");
             
             self.extendedLifetimeAccessToken = accessToken;
+#if !EXCLUDE_FROM_MSALCPP
```

Comment:
That's a lot of preprocessing for a single file. Any way to improve?

Reply:
Functions being called for more than one time are wrapped by the MACROs; functions being called for only one time is still using EXCLUDE_FROM_MSALCPP